### PR TITLE
Add recommend app button

### LINKED ISSUE #76: Recommend App Button - MVP [Dev-28.0]
User story:
**As a** user
**I want** a quick and familiar way to share the app
**So that** I can share it with other people

Acceptance criteria:
_Given_ the issue is shown
_When_ the user reaches the 'more-page'
_Then_ the share app function is located below 'favourites' as clickable text

_Given_ the share app button shown
_When_ the user clicks it
_Then_ a system share dialog is displayed (like the share article feature)

_Given_ the system dialog is displayed
_When_ an app for sharing is clicked
_Then_ a text from the firebase backend is copied to the respective app

_Given_ the share app feature is used
_Then_ the preset text and link to SN are localized according to the set issue

Dependencies:
- #75
- https://github.com/ConstructiveNews/squirrel-news-firebase/issues/3

DoD Checklist:
- [ ] Acceptance criteria are met
- [ ] Test by developer has been done
- [ ] Code is commented if required
- [ ] Code/PR review has been done
- [ ] Feature is documented
- [ ] Internal test by a 2nd Dev or the PO has been done

### PR DESCRIPTION
**Pull request by hstaatsxx**
closes #76

### CODE REVIEW

**Comment by hstaatsxx** [comment]
FYI: After fixing share function #75 this should be tested one more time. Until then please hold PR.

**Comment by olierxleben** [comment]
lgtm

File: src/pages/FavoritesPage.tsx [hunk 1]
```diff
@@ -43,6 +22,27 @@ export const FavoritesPage: FC = () => {
...
+                    articleId: fav.articleId, 
+                    issue: fav.issueId, 
+                    article: data
+                  } as Favorite
+                )
+              })
+            });
+        })
+      ).then((articleData) => {
+        console.log(articleData)
```
Comment (olierxleben) [inline src/pages/FavoritesPage.tsx:42]:
```suggestion
```

File: src/pages/MorePage.tsx [hunk 2]
```diff
@@ -16,25 +16,54 @@ var style: CSSProperties = {
...
+      await Share.share({
+        title: lang === 'en' ? recommend?.labelEN : recommend?.labelDE ,
+        text: valFor(lang, 'share.text'),
+        url: getStoreLink(),
+        dialogTitle: lang === 'en' ? recommend?.labelEN : recommend?.labelDE 
+      })
+
+    } catch (e) {
+      // add Toast Message or somethng else here
+      console.error(typeof e);
```
Comment (olierxleben) [inline src/pages/MorePage.tsx:54]:
```suggestion
      console.error(e);
```
we should know the error thrown.

**Review by olierxleben — APPROVED** [review]
I have two optional things, but nothing that would prevent merging. :)

**Pull request merged**